README.md

```diff
@@ -485,6 +485,51 @@ deleteAll() async {
 ```
 
 
+## Trouble-Shooting Section
+
+The *first* time I tried to change the path of the Flutter folder so that it could be accessed by the terminal without having to use several paths, it **did not work**, every time I closed the terminal Flutter was no longer recognized.</br> 
+
+![Screen Shot 2020-02-10 at 10 35 56](https://user-images.githubusercontent.com/27420533/74142666-41b7fd80-4bf1-11ea-9916-6b16261349aa.png)
+
+Whenever I used *this* command:</br>
+
+
+![Screen Shot 2020-02-10 at 10 15 19](https://user-images.githubusercontent.com/27420533/74141184-5c3ca780-4bee-11ea-89e8-32cf116d304d.png)
+
+Then I started to notice that this command only worked for the current terminal, so it was not **permanent**.
+Even using the command that should change the path permanently Flutter was still **unrecognized**.</br>
+
+This would be the command that would make Flutter access through the terminal permanent:</br>
+
+![Screen Shot 2020-02-10 at 10 33 43](https://user-images.githubusercontent.com/27420533/74142510-fd2c6200-4bf0-11ea-9b97-2111c6353c17.png)
+
+Changing the path to where we would have cloned Flutter's Git repository [PATH_TO_FLUTTER_GIT_DIRECTORY].
+And then using the command: 
+
+![Screen Shot 2020-02-10 at 10 48 55](https://user-images.githubusercontent.com/27420533/74143814-66ad7000-4bf3-11ea-810f-dbe2fdf2fe6f.png)
+
+
+What will help us refresh the terminal.</br>
+But even so, every time I closed and opened the terminal again Flutter was no longer recognized.</br>
+So I thought that maybe the best option would be to go to the document and edit it with the "export PATH" command but inside the document itself other than in the terminal.</br>
+
+So I went to Finder / Macintosh HD / Users / and to my user.
+How is a hidden file or document we must type cmd + shift + .
+
+![Screen Shot 2020-02-10 at 11 21 07](https://user-images.githubusercontent.com/27420533/74145756-83e43d80-4bf7-11ea-94ca-f6be3cfac21b.png)
+
+Then I opened my *.zhs_profile* and typed the command.
+
+![Screen Shot 2020-02-10 at 11 23 10](https://user-images.githubusercontent.com/27420533/74145885-c4dc5200-4bf7-11ea-88da-c29482d386cd.png)
```

Comment:
```suggestion
`export PATH=/Users/m/Documents/flutter/bin:$PATH`

This instructs my terminal environment to export an updated version of the `PATH` environment variable with `flutter/bin` path _prepended_ to the current `PATH` variable. (_i.e. add `flutter/bin` to the `PATH` so that my terminal knows where to find the `flutter` CLI_)
```